src/transformers/models/maskformer/modeling_maskformer.py

```diff
@@ -267,11 +267,11 @@ class MaskFormerForInstanceSegmentationOutput(ModelOutput):
         loss (`torch.Tensor`, *optional*):
             The computed loss, returned when labels are present.
         class_queries_logits (`torch.FloatTensor`):
-            A tensor of shape `(batch_size, num_queries, height, width)` representing the proposed masks for each
-            query.
-        masks_queries_logits (`torch.FloatTensor`):
             A tensor of shape `(batch_size, num_queries, num_labels + 1)` representing the proposed classes for each
             query. Note the `+ 1` is needed because we incorporate the null class.
```

Comment:
```suggestion
            A tensor of shape `(batch_size, num_queries, num_labels + 1)` representing the class logits for each
            query. Note the `+ 1` is needed because we incorporate the null class.
```